### ISSUE TITLE
bug: bang dquote is special in zsh

### ISSUE THREAD

**Issue by MikeSpreitzer** [kubestellar/kubestellar#1177]
### Describe the bug

Our documentation tells people to copy-and-paste some commands that include `!"`.  But that is special to zsh.

```
mspreitz@mjs13 kubestellar % echo $0
-zsh

mspreitz@mjs13 kubestellar % echo "Hey! "
Hey! 

mspreitz@mjs13 kubestellar % echo "Hey!"
dquote> 
```

There are four instances of this problem in `docs/content/Coding Milestones/PoC2023q1/example1-subs/example1-stage-4.md`

There are also `!"` appearing in some scripts, but those are all scripts that go specifically to bash.

### Steps To Reproduce

See above

### Expected Behavior

A user of a common shell can copy-and-paste what our docs offer.

**Comment by MikeSpreitzer** [comment]
squotes are different.

```
mspreitz@mjs13 kubestellar % echo 'Hey!'
Hey!
```

**Comment by clubanderson** [comment]
fixed in main and 0.9.0 version of doc
